### PR TITLE
Document early access journey designs

### PR DESCRIPTION
**Pull request by hazalarpalikli**
# PR Checklist

- [ ] If you are proposing a new decision record document, used the right template for that
      - ([ADR](https://github.com/alphagov/forms/blob/main/ADR/ADRXXX-architecture-decision-record-template.md), [decision-record](https://github.com/alphagov/forms/blob/main/decision-record/DRXXX-decision-record-template.md), engagement, [research](https://github.com/alphagov/forms/blob/main/research/YYYY-MM-DD-template.md))
- [ ] Set yourself as the Assignee
- [ ] Tag anyone you would like to review, or @forms-design or @forms-devs
- [ ] Fill in the template below

---

# What

Describe what you have changed and why.

# How to review

Describe the steps required to test the changes.

For example:

1. Semantic: Do you agree with the changes?
1. Syntactic: Spelling, grammar, etc.

# Who can review

Describe who can review the changes. Or more importantly, list the people
that can't review, because they worked on it.

### CODE REVIEW

File: design/features/early access journey [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Editing pages when looking at a preview of a form
```
Comment (hannahkc) [inline design/features/early access journey:1]: Think this is the wrong heading?

File: design/features/early access journey [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+The changes are in the product. 
+Prototype can be found here: https://forms-prototypes-pr-201.herokuapp.com/product-pages/ 
+
+<br>
+
+___
+
+<br>
+
+[Back to the top](#editing-pages-when-looking-at-a-preview-of-a-form)
```
Comment (hannahkc) [inline design/features/early access journey:49]: think this is the wrong link?

File: design/features/early access journey (hunk 1 above)
Comment (hazalarpalikli) [reply design/features/early access journey:1]: Yeah i need to delete this

File: design/features/early access journey (hunk 2 above)
Comment (hazalarpalikli) [reply design/features/early access journey:49]: It is the right link, i have added the protoype to the forms protoype.

Comment (hannahkc) [reply design/features/early access journey:49]: meant this: 

[Back to the top](#editing-pages-when-looking-at-a-preview-of-a-form)

which won't work when you correct the heading.

Comment (hannahkc) [reply design/features/early access journey:49]: Also I think the idea with this is to record designs here so we can look back at previous versions and why they were the way they were. The prototype is only temporary and can be replaced, deleted, moved. We should only be using the prototype for testing things really.

**Pull request closed without merging**